### ISSUE TITLE
Add ukrainian localization

### ISSUE THREAD

**Issue by yevgmikh** [capnkirok/animaniamod#558]
I think I can make Ukrainian translation. Do I need create pull request to add .lang-file or I can attach the file here and the repository members will add the localization file?
All .lang-files located [here](https://github.com/capnkirok/animaniamod/tree/1.12/src/main/resources/assets/animania/lang)?(There are no other places where there are files to be translated, right?)
Joined to discord server as Call me senpai#5935

**Comment by Tschipp** [comment]
a PR would be great

**Comment by yevgmikh** [comment]
Ok. Are there any other places where located .lang-files except [src/main/resources/assets/animania/lang](https://github.com/capnkirok/animaniamod/tree/1.12/src/main/resources/assets/animania/lang)?

**Comment by Tschipp** [comment]
Yes, https://github.com/capnkirok/animaniamod/blob/1.12/src/main/resources/assets/catsdogs/animania/lang/en_us.lang

**Comment by yevgmikh** [comment]
Translation done. What branches use in pull request?

**Comment by yevgmikh** [comment]
Pull requert (#[animaniamod-558](https://github.com/capnkirok/animaniamod/pull/559)) is awaiting merge.
